package.json

```diff
@@ -5,7 +5,7 @@
   "main": "index.js",
   "scripts": {
     "build": "webpack",
-    "start": "webpack-dev-server"
+    "start": "webpack-dev-server webpack.common.js"
```

Comment:
You should have a dedicated `prod` file and a dedicated `dev` file. The `common` file should not serve as the entry file for the configuration, but instead it should be used by the `dev` and `prod` configuration files.